content/kubermatic/main/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -344,85 +383,188 @@ spec:
-- The updated `VirtualMachineInstancePreset` will be reconciled from the `default` namespace into the `cluster-xyz` namespace.
-  Give it some time to be reconciled. The reconciliation interval is configurable (refer to `providerReconciliationInterval`
-  in [Seed configuration]({{< ref "../../../tutorials-howtos/project-and-cluster-management/seed-cluster/" >}})
-- For all `VirtualMachineIsntances` already created, this will have no impact.
-  Please refer to [KubeVirt Preset documentation](https://kubevirt.io/user-guide/virtual_machines/presets/#updating-a-virtualmachineinstancepreset)
-- The update will then be effective for new `VirtualMachineIsntances`.
+{{< tabs name="Kubermatic instancetypes" >}}
 
-*Note 2:* Limitation of the list of reconciled fields.
+{{% tab name="standard-2 instancetype" %}}
+```yaml
+apiVersion: instancetype.kubevirt.io/v1alpha1
+kind: VirtualMachineInstancetype
+metadata:
+  name: standard-2
+spec:
+  cpu:
+    guest: 2
+  memory:
+    guest: 8Gi
+```
+{{% /tab %}}
+{{% tab name="standard-4 instancetype" %}}
+```yaml
+apiVersion: instancetype.kubevirt.io/v1alpha1
+kind: VirtualMachineInstancetype
+metadata:
+  name: standard-4
+spec:
+  cpu:
+    guest: 4
+  memory:
+    guest: 16Gi
+```
+{{% /tab %}}
+{{% tab name="standard-8 instancetype" %}}
+```yaml
+apiVersion: instancetype.kubevirt.io/v1alpha1
+kind: VirtualMachineInstancetype
+metadata:
+  name: standard-8
+spec:
+  cpu:
+    guest: 8
+  memory:
+    guest: 32Gi
+```
+{{% /tab %}}
+{{% tab name="socket-advantage preference" %}}
+```yaml
+apiVersion: instancetype.kubevirt.io/v1alpha1
+kind: VirtualMachinePreference
+metadata:
+  name: sockets-advantage
+spec:
+  cpu:
+    preferredCPUTopology: preferSockets
+```
+{{% /tab %}}
+{{< /tabs >}}
 
-Please note that not all fields of the `VirtualMachineInstance` are merged into the `VirtualMachineInstance`.
-[](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-api/webhooks/mutating-webhook/mutators/preset.go#L123)
-For the `domain`, only the following fields are merged:
-- `CPU`
-- `Firmware`
-- `Clock`
-- `Features`
-- `Devices.Watchdog`
-- `IOThreadsPolicy`
+#### How to provide your own **Custom** instancetypes and preferences
 
-*Note3:* Migration to the instanceType API
 
-Please note that in the next KKP release, we will migrate to the new *VirtualMachineInstancetype* ([alpha release](https://github.com/kubevirt/api/blob/main/instancetype/v1alpha1/types.go#L35)),
-as the *VirtualMachineInstancePreset* will be deprecated. This will for example lift the limitation around the list of
-merged fields from the `VirtualMachineInstance` and provide deterministic behavior with immediate detection of merge conflicts.
+Just create some `VirtualMachinClusterInstancetype` and/or some `VirtualMachineClusterPreference` (custer-wide resources) in the KubeVirt infrastructure cluster and you are can use them to template your VMs.
```

Comment:
small typo.
`custer-wide` should be `cluster-wide`

Reply:
done